__TESTS__/unit/url/createCloudinaryURL.test.ts

```diff
@@ -1,8 +1,50 @@
 import createCloudinaryURL from "../../../src/url/cloudinaryURL";
+import {minimumPad, crop, fill, scale} from '../../../src/actions/resize/Resize';
```

Comment:
Can we add to this PR the TransformableImage.getUrlForImage implementation?

getUrlForImage should call createCloudinaryURL internally and accept publicID and descriptor